examples/seismic/model.py

```diff
@@ -52,7 +74,37 @@ def demo_model(preset, **kwargs):
         return Model(vp=v, origin=origin, shape=shape,
                      spacing=spacing, nbpml=nbpml, **kwargs)
 
-    elif preset.lower() in ['circle']:
+    elif preset.lower() in ['layerstti', 'twolayertti', '2layertti']:
```

Comment:
consistency with the -isotropic names: put a '-' in all names right before 'tti'